static/app/components/searchSyntax/renderer.tsx

```diff
@@ -218,21 +218,32 @@ const InList = styled('span')`
   }
 `;
 
-const LogicGroup = styled('span')`
-  &:before,
-  &:after {
+const LogicGroup = styled(({children, ...props}) => (
+  <span {...props}>
+    <span>(</span>
+    {children}
+    <span>)</span>
+  </span>
+))`
+  > span:first-child,
+  > span:last-child {
     position: relative;
-    font-weight: bold;
-    color: ${p => p.theme.white};
-    padding: 3px 0;
-    background: ${p => p.theme.red200};
-    border-radius: 1px;
+    color: transparent;
+
+    &:before {
+      position: absolute;
+      top: -5px;
+      color: ${p => p.theme.orange400};
+      font-size: 16px;
```

Comment:
```suggestion
      font-size: ${p => p.theme.fontSizeLarge};
```

Reply:
I'm going to avoid using our theme defined font size here actually, since we are specially aligning things based on font size, so I think I would prefer (for now at least) to be more explicit.

May revisit in the future